tests/integration/foglamp/services/core/test_scheduler.py

```diff
@@ -24,9 +24,7 @@
 
```

Comment:
this file should move to unit?

Reply:
This will be addressed when dependencies of running foglamp is removed and the test is a pure unit test.